web-next/src/routes/(root)/search.tsx

```
@@ -0,0 +1,271 @@
+      void loadSearchPostsQuery(
+        query,
+        i18n.locale,
+        i18n.locales != null && Array.isArray(i18n.locales) ? i18n.locales : [],
+      );
+    } else if (searchType === "handle" || searchType === "url") {
+      void loadSearchObjectQuery(query);
+    }
+  },
+} satisfies RouteDefinition;
+
+const searchPostsPageQuery = graphql`
```

Biome (2.1.2)

[error] Shouldn't redeclare 'searchPostsPageQuery'. Consider to delete it or rename it.

'searchPostsPageQuery' is defined here:


(lint/suspicious/noRedeclare)